tutorials/java/CurrentPlaceDetailsOnMap/README.md

```diff
@@ -7,7 +7,7 @@ This sample goes hand in hand with a tutorial for the Google Maps Android API:
 Prerequisites
 --------------
 
-- Android SDK v31
+- Android SDK v33
```

Comment:
If pre-req should match minSDK, this should be 19

Reply:
Check all READMEs for minSDK match

Reply:
This should match the `compileSdk `(i.e., the value specified in `compileSdkVersion `must be locally installed, and it is hence a requirement). I am reviewing it for other projects.